src/network/networkprotocol.h

```diff
@@ -620,6 +622,12 @@ enum ToClientCommand
 		u32 id
 	*/
 
+	TOCLIENT_NODEMETA_CHANGED = 0x54,
+	/*
+		u16 command
```

Comment:
nrz wants to not document the command field, as it exists in any packet (there is no extra information).